homeassistant/components/config/entity_registry.py

```diff
@@ -106,6 +106,11 @@
 
     if 'new_entity_id' in msg:
         changes['new_entity_id'] = msg['new_entity_id']
+        if (msg['new_entity_id'] in hass.states.async_entity_ids(
```

Comment:
Instead of doing the check like this, it's faster to just do `if hass.states.get(msg['new_entity_id']) is not None:`